### PR TITLE
Install go via gimme

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

**Comment by fedepaol** [comment]
You also need what's being done in https://github.com/openshift-kni/cnf-features-deploy/pull/46 (hoping it's the last one)

**Comment by cynepco3hahue** [comment]
> You also need what's being done in [openshift-kni/cnf-features-deploy#46](https://github.com/openshift-kni/cnf-features-deploy/pull/46) (hoping it's the last one)

Yep already added it, let see if it will work:)

File: openshift-ci/Dockerfile.tools [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-ENV PATH=$PATH:$GOPATH/bin
-ENV GOVERSION golang-1.13.5-1.fc31
+ENV GOVERSION 1.13.5
+ENV PATH=$PATH:/root/.gimme/versions/go"$GOVERSION".linux.amd64/bin:$GOBIN
 ARG GO_PACKAGE_PATH=github.com/openshift-kni/performance-addon-operators
 
 # rpms required for building and running test suites
 RUN dnf -y install \
-    $GOVERSION \
+    git \
```
Comment (slintes) [inline openshift-ci/Dockerfile.tools]: can you add gcc please? That should be it then.

Comment (cynepco3hahue) [reply openshift-ci/Dockerfile.tools]: done

**Pull request merged**